app/backend/approaches/chatreadretrieveread.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         self.content_field = content_field
         self.query_language = query_language
         self.query_speller = query_speller
-        self.chatgpt_token_limit = get_token_limit(chatgpt_model)
+        self.chatgpt_token_limit = get_token_limit(chatgpt_model, default_to_minimum=self.ALLOW_NON_GPT_MODELS)
```

Comment:
We might want to update the docs to say that we can't estimate tokens in this case

Reply:
One question - does it even try to get the token limit if default_to_minimum is True? It seems to be always true even if we aren't using a non-gpt model

Reply:
https://github.com/pamelafox/openai-messages-token-helper/blob/eb59b9a202505720bc2f1db7dadf1b78500eb114/src/openai_messages_token_helper/model_helper.py#L42-L58

Reply:
looks like it does

Reply:
default_to_minimum is False unless set otherwise, and you'd get an error with a non-recognized model:
https://github.com/pamelafox/openai-messages-token-helper/blob/eb59b9a202505720bc2f1db7dadf1b78500eb114/src/openai_messages_token_helper/model_helper.py#L42

Reply:
Updated docs!